content/blog/2019/10/2019-10-30-jenkins-performance-avoiding-pitfalls.adoc

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Jenkins Performance: Avoiding Pitfalls, Diagnosing Issues, and Scaling for Growth - DevOps World | Jenkins World 2019"
+tags:
+- jenkins world
+- jenkinsworld
+- devopsworld2019
+- performance
+authors:
+- ryansmith303
+opengraph:
+  image: /images/performance/gc-unhealthy.png
+  image: /images/logos/jenkenstein/256.png
+---
+
+NOTE: This is a speaker blogpost for a DevOps World | Jenkins World 2019 talk in Lisbon, Portugal
+
+With Halloween upon us, there's no better time to discuss Jenkinstein! Are you suffering from Jenkins Performance issues? Are users complaining about a slow UI or even scarier, Jenkins is frequently going down?  
```

Comment:
I would say, "even scarier, Is Jenkins going down frequently?" Reads a bit better.